components/dashboard/src/Usage.tsx

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
This file is almost identical to `src/teams/TeamUsage.tsx`. Can we use one in both places?

Reply:
@andrew-farries, I think there are two options, having the entry points separate (like now) and computing the props depending without consideration of location, or we merge, and need to sort out, what's meaningful inside of a single component depending on the location. 
I'd prefer to go with the current state, just to avoid the complexity of properly creating the props for `<UsageView />`.